Sources/Model/GraphQL/Query/HyperwalletTransferMethodUpdateConfigurationQueries.swift

```diff
@@ -0,0 +1,107 @@
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+import Foundation
+
+/// The `HyperwalletTransferMethodUpdateConfigurationFieldQuery`
+/// struct defines and builds a query to retrieve the fields
+/// required to update a transfer method (Bank Account, Bank Card, PayPal Account, Prepaid Card, Paper Check)
+/// with the Hyperwallet platform.
+public struct HyperwalletTransferMethodUpdateConfigurationFieldQuery: GraphQlQuery, Hashable {
+    private var transferMethodToken: String
+    private var query = """
+        query QueryUpdateTransferMethod(
+            $trmToken: String =  "%@"
+        ){
+            transferMethodUpdateUIConfigurations (
+                trmToken: $trmToken
+            ) {
+                nodes {
+                    country
+                    currency
+                    transferMethodType
+                    profile
+                    fieldGroups {
+                        nodes {
+                            group
+                            isEditable
+                            fields {
+                                category
+                                value
+                                dataType
+                                isRequired
+                                isEditable
+                                label
+                                maxLength
+                                minLength
+                                name
+                                placeholder
+                                regularExpression
+                                fieldSelectionOptions {
+                                    label
+                                    value
+                                }
+                                validationMessage {
+                                    length
+                                    pattern
+                                    empty
+                                }
+                                fieldValueMasked
+                                mask {
+                                    conditionalPatterns {
+                                        pattern
+                                        regex
+                                    }
+                                    defaultPattern
+                                    scrubRegex
+                                }
+                            }
+                        }
+                    }
+                    bankBranches {
+                        nodes {
+                            bankCode
+                            branches {
+                                value
+                                label
+                            }
+                        }
+                        bankElement
+                        branchElement
+                    }
```

Comment:
no need of this as well in the query since we are not using it anywhere

Reply:
Removed the used details.